src/App.js

```diff
@@ -10,7 +10,7 @@ import Spinner from './Spinner'
 
 export default function App () {
   const [files, setFiles] = useState([])
-  const [chunker, setChunker] = useState('size-512')
```

Comment:
I thought this was intentionally set small so the user didn't have to upload a massive file before it would start making interesting DAGs?

Reply:
Yes I seem to remember that was the intent.

Reply:
I'll change it back, I initially bumped it up to avoid network spamming while I was updating things and before I shut off net traffic